### ISSUE TITLE
Add callsign

### ISSUE THREAD

**Issue by fedorcomander** [me-no-dev/KissKONG#1]
Can you do that?

**Comment by me-no-dev** [comment]
yes, but that has to manually configured at build. Maybe I'll think of something better.

**Comment by me-no-dev** [comment]
@fedorcomander I'm kinda still waiting for that 1.2 to get to my email ;) I went out today to fly a few packs and I'm coming home all kinds of unhappy. Could you please connect me to a firmware developer? I have some issues to report on the latest Dshot that I was able to get to (1.1e RC9) and have some video to show the problem and I have a sad fail on AIO v2.
If you are unwilling to deal with me, please do just say so and save me the time and embarrassment to keep trying to contact you.

**Comment by fedorcomander** [comment]
1.2 is released yesterday. for esc dev - you can contact juuut (Felix) on rig or flyduino support.
